### PR TITLE
Really random fixlets

### PR DESCRIPTION
**Pull request by keszybz**

### CODE REVIEW

File: src/oom/oomd-manager.c [hunk 1]
```diff
@@ -108,10 +108,7 @@ static int process_managed_oom_message(Manager *m, uid_t uid, JsonVariant *param
                 if (streq(message.property, "ManagedOOMMemoryPressure") && message.limit > 0) {
                         int permyriad = UINT32_SCALE_TO_PERMYRIAD(message.limit);
 
-                        r = store_loadavg_fixed_point(
-                                        (unsigned long) permyriad / 100,
-                                        (unsigned long) permyriad % 100,
-                                        &limit);
+                        r = store_loadavg_fixed_point(permyriad / 100LU, permyriad % 100LU, &limit);
```
Comment (bluca) [inline src/oom/oomd-manager.c:111]: eheh I see you found where I cargo-culted from for my other PR :-D

File: src/test/test-sizeof.c [hunk 2]
```diff
@@ -93,5 +93,13 @@ int main(void) {
 
         printf("timeval: %zu\n", sizeof(struct timeval));
         printf("timespec: %zu\n", sizeof(struct timespec));
+
+        void *x = malloc(100);
+
+        printf("local variable: %p\n", &function_pointer);
+        printf("glibc function: %p\n", memcpy);
+        printf("heap allocation: %p\n", x);
```
Comment (bluca) [inline src/test/test-sizeof.c:101]: if malloc fails, will printf complain that x is null?

**Review by bluca — APPROVED** [review]
lgtm

File: src/test/test-sizeof.c (hunk 2 above)
Comment (keszybz) [reply src/test/test-sizeof.c:101]: It's just printing the pointer value, and 0 should be a completely valid value. FWIW, glibc prints `(nil)`, I don't know if this is standarized somehow.

File: src/oom/oomd-manager.c (hunk 1 above)
Comment (keszybz) [reply src/oom/oomd-manager.c:111]: Ack ;)

**Pull request merged**